crypto/engine/eng_linux_getrandom.c

```diff
@@ -0,0 +1,150 @@
+#include <openssl/opensslconf.h>
+
```

Comment:
Well, maybe because nobody knew about the new mailing list. Well, I didn't know about it anyway :P

In any case this seems the wrong way to add `getrandom()` support. It should be added to `RAND_poll()` instead
https://github.com/openssl/openssl/blob/master/crypto/rand/rand_unix.c#L242

which is in dire need of a refactoring btw like what I proposed in (and previously in that big mailing list discussion that didn't go anywhere):
https://github.com/openssl/openssl/issues/898#issuecomment-199361100

but yes, it's probably too late at this point for 1.1.0.

RNG-related is also https://github.com/openssl/openssl/pull/512 which is not a new feature.